code/product.json

```diff
@@ -80,6 +80,11 @@
 				},
 				"publisherDisplayName": "Microsoft"
 			}
+		},
+		{
+			"name": "devfile.vscode-devfile",
+			"version": "0.0.2",
```

Comment:
would be nice to understand why `latest` does not work

Reply:
yes, using `latest` here would be the best solution

added comment here https://github.com/eclipse/che/issues/21606#issuecomment-1921457345